src/pages/UsersPage.js

```diff
@@ -36,11 +36,14 @@ export default function UsersPage({ users, goToCreatepage }) {
 }
 
 const Wrapper = styled.section`
-  display: flex;
-  flex-direction: column;
-  gap: 10px;
   list-style-type: none;
-  max-width: min-content;
+  height: 100vh;
```

Comment:
you give the UsersPage 100vH but the App already has 100vH as well. I am not 100% sure but I think this is not the best solution

Reply:
I am assuming if you give the App a 100vH with the grid-template-rows: auto 60px. The navigation will take in the 6opx at the bottom and the pages can be set to height 100% and they should take in the rest of the site.